pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -632,7 +661,7 @@ func buildAttachedBinding(binding *workv1alpha2.ResourceBinding, object *unstruc
 
 	return &workv1alpha2.ResourceBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      names.GenerateBindingName(object.GetKind(), object.GetName()),
+			Name:      names.GenerateWorkName(object.GetKind(), object.GetName(), object.GetNamespace()),
```

Comment:
This make new name style for rb, how to handle existing old name style rb? This make dumplicate rb for same reource.

Reply:
Sorry, it's my mistake, thanks for reminding